### PR TITLE
Test subsampling for calculating PS throws no error

### PR DESCRIPTION
**Pull request by aki-nishimura**
I didn't have push permission to @azimov's fork, so am making a pull request from my own fork. 

The test just makes sure that no error is thrown when `maxCohortSizeForFitting` is set to a value less than either target or control population size.

### CODE REVIEW

File: tests/testthat/test-ps_subsampling.R [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  expect_true(min(targetPopSize, comparatorPopSize) > 10)
+
+  expect_error(
+    createPs(
+      cohortMethodData, population,
+      errorOnHighCorrelation = FALSE,
+      maxCohortSizeForFitting = 10,
+      control = cyclops_control
+    ),
+    NA
```
Comment (azimov) [inline tests/testthat/test-ps_subsampling.R:42]: Is a specific error message thrown that can be set here?

**Comment by azimov** [comment]
applogies @aki-nishimura I forgot to manage access to the fork today. I've now sent you an invite.

File: tests/testthat/test-ps_subsampling.R [hunk 2]
```diff
@@ -39,7 +39,7 @@ test_that("Subsampling cohort throws no error", {
       maxCohortSizeForFitting = 10,
       control = cyclops_control
     ),
-    NA
+    NA # "NA" asserts that there should be no errors
```
Comment (aki-nishimura) [inline tests/testthat/test-ps_subsampling.R:42]: @azimov I am not sure if I understood your previous comment, but I think this additional comment would help?

Comment (azimov) [reply tests/testthat/test-ps_subsampling.R:42]: I follow,  this makes it clearer

**Review by msuchard — APPROVED** [review]

**Pull request merged**